cypress/integration/movie-details-spec.js

```diff
@@ -0,0 +1,93 @@
+let movie4 = { "movie": { "id": 737173 } }
+let videos4 = { "videos": [] }
```

Comment:
I included these two, not as fixtures, because it seemed silly to make two files just to test a lack of trailers.